### PR TITLE
update metallb manifests

### PR DESCRIPTION
**Pull request by alknopfler**
- update the metallb manifests to reduce complexity and fix the charts link

### CODE REVIEW

**Review by atanasdinov — APPROVED** [review]

File: telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/telco-capi-metallb-multi-node.yaml [hunk 1]
```diff
@@ -74,90 +74,68 @@ spec:
...
-                User=root
-                ExecStartPre=/bin/sh -c "while [ ! -f /var/lib/rancher/rke2/bin/kubectl ] && [ ! -f /etc/rancher/rke2/rke2.yaml ]; do sleep 1; done"
-                ExecStart=/bin/sh -c "while [ $(/var/lib/rancher/rke2/bin/kubectl --kubeconfig=/etc/rancher/rke2/rke2.yaml get deployment --ignore-not-found metallb-controller | wc -l) -eq 0 ]; do sleep 1; done"
-                ExecStart=/bin/sh -c "/var/lib/rancher/rke2/bin/kubectl --kubeconfig=/etc/rancher/rke2/rke2.yaml rollout status deployment metallb-controller -n default --timeout=150s"
-                ExecStart=/bin/sh -c "cp /var/metallb-cr.yaml /var/lib/rancher/rke2/server/manifests/"
-                ExecStartPost=/bin/sh -c "cp /var/endpoint-svc.yaml /var/lib/rancher/rke2/server/manifests/"
-                [Install]
-                WantedBy=multi-user.target
         storage:
           files:
```
Comment (e-minguez) [inline telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/telco-capi-metallb-multi-node.yaml:78]: I'd say you better add a folder creation here as well:

```
        storage:
          directories:
            - path: /var/lib/rancher/rke2/server/manifests
              overwrite: true
```

Comment (alknopfler) [reply telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/telco-capi-metallb-multi-node.yaml:78]: but the rke2-capi-provider has to be created it before (the ignition is executed after the postCommands...) so the folder should be always present before starting this part. 
BTW I've tested on the lab and it worked as expected:
```
localhost:/var/lib/rancher/rke2/server/manifests # ls
endpoint-copier-operator.yaml  metallb.yaml	  rke2-ingress-nginx.yaml	     rke2-snapshot-controller.yaml
endpoint-svc.yaml	       rke2-canal.yaml	  rke2-metrics-server.yaml	     rke2-snapshot-validation-webhook.yaml
metallb-cr.yaml		       rke2-coredns.yaml  rke2-snapshot-controller-crd.yaml
```

Comment (e-minguez) [reply telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/telco-capi-metallb-multi-node.yaml:78]: Ok, I mean, it won't hurt if present but if it is not needed that's fine.

**Review by e-minguez — APPROVED** [review]
LGTM (in fact this is what I'm using right now)

**Pull request merged**